### ISSUE TITLE
Stack already fitted models?

### ISSUE THREAD

**Issue by jmrichardson** [vecxoz/vecstack#44]
Hi, is it possible to stack already fitted models?  I can't find reference in documentation.  Essentially, I have multiple fitted models (based on different X transformations with ML and DL.  Thanks in advance

**Comment by vecxoz** [comment]
Hi,

There is no API to load models which were trained outside vecstack.

Looking a bit deeper I want to note that if your models were trained on **full** train set without k-fold procedure it is impossible to get correct transformation of train set (i.e. `S_train`) because predictions for train set will be overfitted. In this case you can do the following: predict test set using each of your models and compute weighted average of predictions. This result will be close to what stacking could give you.
